src/styles/reset.scss

```diff
@@ -0,0 +1,10 @@
+* {
+  box-sizing: border-box;
+  margin: 0;
+  padding: 0;
+  font-family: "Roboto", sans-serif;
```

Comment:
font-family move to html tag
reset margin and paddings only for those tags you need
```suggestion
```